src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -52,6 +52,7 @@ type ControllerConfig struct {
 	CACertPath           string `envconfig:"CA_CERT_PATH" required:"false" default:""`
 	Namespace            string `envconfig:"NAMESPACE" required:"false" default:"assisted-installer"`
 	OpenshiftVersion     string `envconfig:"OPENSHIFT_VERSION" required:"true"`
+	CheckClusterVersion  bool   `envconfig:"CHECK_CLUSTER_VERSION" required:"false" default:"false"`
```

Comment:
I think the name is a little bit confusing and not clear - maybe it should be called: WAIT_FOR_CLUSTER_VERSION

Reply:
np problem, I'll change that.